### ISSUE TITLE
Grid system proposal

### ISSUE THREAD

**Issue by wesoudshoorn** [appsignal/design-language#21]
I'm proposing a grid system based on a base grid of 8 pixels.

**A full layout would have**
- 12 columns of 72 pixels
- Gutters of 24 pixels
- Margins left and right of 32 pixels

![image](https://user-images.githubusercontent.com/1215495/62939343-56960e80-bdd1-11e9-8539-276dc31b5b8e.png)

**Comment by adamyeats** [comment]
Great! This is excellent. Next steps:

- Put these inside tokens (Does Theo provide a mechanism for this? Check the Salesforce example)
- Export any assets and artefacts e.g. templates, settings files for Figma/Sketch. These can potentially live inside this repo.

**Comment by wesoudshoorn** [comment]
@xademy I've put these inside of design tokens. There's no default system, so I've added them in the spacing category.

As for the assets / artefacts, I just upgraded to a paid plan on Figma for the two of us, so we can add "team libraries". We're now able to push styles to a team level instead of document level. I have not found a way to export base templates/styles etc. that could live in this repo.
